build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -35,16 +36,28 @@
     <Rule Id="CA1720" Action="None" />
     <Rule Id="CA1721" Action="None" />
     <Rule Id="CA1724" Action="None" />
+    <Rule Id="CA1801" Action="None" />
+    <Rule Id="CA1802" Action="None" /> <!-- disabled due to lack of support for string literals -->
+    <Rule Id="CA1804" Action="None" />
+    <Rule Id="CA1806" Action="None" />
+    <Rule Id="CA1812" Action="None" /> <!-- disabled as we create many internal types via reflection-based mechanisms -->
```

Comment:
@jinujoseph @kuhlenh This comment shows what I expect to be a common case in the wild